content/project-organization/quality-council/processes/pr-management.md

```diff
@@ -0,0 +1,159 @@
+
+![img.png](images/readme8.png)
+
+
+We can see that in **violet** we've put the "**Branch "** we're interested in and in **brown** the name of our file, which can be modified as we wish. 
+
+Still with this **Branch**, I advise you to launch the automated Tests. To do this, I suggest you look at the end of this chapter dedicated to it.
+
+
+### 2. Replicating the issue
+
+Now that we have an installed shop in which the issue is still present, you can check the three elements of the table: 
+
+- Description: Sometimes, the description will tell you where the PR is heading, but it can also tell you where it may have affected the code and which axes we'll be able to check next.
+- How to Test: When this field is filled in, it enables you to easily identify a way of reproducing the issue. Often, it will simply refer to the issue linked to the PR.
+- Fixed issue or discussion: When you click on the indicated issue, you'll have all the information you need to reproduce it.
+
+Once on the issue, all you have to do is scroll down to find the "Steps to reproduce" section and follow the steps one by one, taking the example of our same PR:
+
+![img.png](images/readme9.png)
+
+If the issue has been validated by our testers, ( ⚠️ any PR that mentions an issue must have this issue validated by the QA, otherwise the PR will not be taken into account ) the steps to reproduce must have been reviewed by them and is therefore easily reproducible. 
+
+ℹ️ If the issue has been validated by a Developer, the PR in question must be reviewed by a Dev, unless otherwise specified. 
+ℹ️ If you have installed your shop in *Computer/var/www/html,* you will be able to use your shop from a web browser and put the link : *localhost/PR_81x*
+
+### 3 Installing the PR
+
+
+Once we've reproduced the issue, we now need to check that the PR actually fixes the problem. To do this, we first need to install it in our shop. We'll take out the terminal in the **PR_81x** folder and you'll be able to type these commands: 
+
+
+![img.png](images/readme10.png)
+
+
+For the color code, we always use **violet** for the **Branch** and this time we'll use **green** for the **PR number**. 
+
+
+We can find this **number** in the title of the PR, as you can see. 
+
+
+![img.png](images/readme11.png)
+
+
+All you have to do now is try to reproduce the issue again to see if the PR corrects it correctly.
+
+### 4. Check that the PR doesn't create another issue
+
+Now that you've checked that the PR has corrected the issue, you need to make sure that it doesn't create another one instead. To do this, you need test axes. 
+
+To find them, go to the Files changed tab: 
+
+![img.png](images/readme12.png)
+
+You'll then have all the changes made to the PR. To do this, you'll get this view: 
+
+![img.png](images/readme13.png)
+
+Seen like this, it may seem rather difficult to see the differences and what might be impacted, so I advise you to modify the split view and save.
```

Comment:
```suggestion
Seen like this, it may seem rather difficult to see the differences and what might be impacted, so we advise you to modify the split view and save.
```